### ISSUE TITLE
Pytorch to ONNX export: Unexpected reduction in model size.

### ISSUE THREAD

**Issue by Ashish0091** [onnx/onnx#5828]
I am attempting to convert a PyTorch text detection model to ONNX, but I am experiencing unexpected behavior during the export process. The size of the exported ONNX model is significantly reduced from the original PyTorch model (15 MB to 7 MB), and the inferences from the ONNX model are not as accurate as the PyTorch model.

### Environment
- PyTorch Version: 1.12.0
- ONNX Version:  1.12.0
- Operating System: Ubuntu

I used the mentioned repo to train a text detection model
Model name: EAST text detection model [https://github.com/mingliangzhang2018/EAST-Pytorch](url)

What I have tried: 
I have tried using different PyTorch and ONNX versions, but the issue remains.
To isolate the problem, I tested the ONNX export with the tutorial [https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html?highlight=onnx](url) and it did not exhibit size reduction issues.
Also I have tried different versions of torch (1.14) and onnx but still have the same issue. During the conversion from the torch, there are no errors or warnings.

**Comment by justinchuby** [comment]
I suggest using the latest PyTorch-nightly build to see if you can reproduce this issue, as more fixes have been merged in the latest version. For further questions about the PyTorch converter I encourage opening issues in the PyTorch repository for more help. Be sure to include a code snippet that will reproduce what’s mentioned above (size reduction, accuracy). Thanks and happy New Year!
